evcache-client/src/main/java/com/netflix/evcache/pool/eureka/EurekaNodeListProvider.java

```diff
@@ -56,7 +56,14 @@ public EurekaNodeListProvider(ApplicationInfoManager applicationInfoManager, Eur
      */
     @Override
     public Map<ServerGroup, EVCacheServerGroupConfig> discoverInstances(String _appName) throws IOException {
-        if ((applicationInfoManager.getInfo().getStatus() == InstanceStatus.DOWN)) {
+        final Property<Boolean> ignoreAppEurekaStatus = props.get("evcache.ignoreAppEurekaStatus", Boolean.class).orElse(false);
+
+        if (ignoreAppEurekaStatus.get())
+            log.info("Not going to consider the eureka status of the application, to initialize evcache client.");
+
+        if (!ignoreAppEurekaStatus.get() && (applicationInfoManager.getInfo().getStatus() == InstanceStatus.DOWN)) {
+            log.info("Not initializing evcache client as application eureka status is DOWN. " +
+                    "One can override this behavior by setting evcache.ignoreAppEurekaStatus on application.");
```

Comment:
Also maybe include value for the property, like
"...by setting evcache.ignoreAppEurekaStatus property to true, scoped to your application."